internal/cli/cmd/version.go

```diff
@@ -21,14 +21,26 @@ import (
 	"fmt"
 
 	"github.com/urfave/cli/v2"
+
+	"github.com/suse/elemental/v3/internal/version"
```

Comment:
The initial idea in mind was for the `cmd` package to never import anything but `urfave/cli` types and functions. This way it seemed cleaner to me. Do we benefit from inlining the action in this module?

Reply:
Resolved in latest version (removed `internal/version` package)